classes/game.rb

```diff
@@ -0,0 +1,20 @@
+require './item'
+require 'date'
+
+class Game < Item
+  attr_accessor :last_played_at, :multiplayer
+  attr_reader :publish_date
+
+  def initialize(multiplayer, last_played_at, publish_date)
+    super(publish_date)
+    @multiplayer = multiplayer
+    @last_played_at = last_played_at
+  end
+
+  private
+
+  def can_be_achived?(date)
+    true if super.publish_date && @last_played_at > 2
```

Comment:
- [ ] To access the parent function you should only call super, using super.publish_date will fail.

- [ ] Please remove the argument since you are not using it (date)

Reply:
Alright, bro. Thanks a lot